src/state/index.js

```diff
@@ -1,5 +1,5 @@
 import { combineReducers, createStore, applyMiddleware } from 'redux';
```

Comment:
we should think about using `configureStore` from redux toolkit if its accessible... it comes baked in with some default middleware (including thunk). Also sets up redux devtools which we are currently missing and I would love to have.

Reply:
Thanks Chaz, I'm unfamiliar with configureStore and Chris had setup this portion of the code! Let's get his thoughts on this too but I defer to your expertise on this :)